### PR TITLE
Align credentials stores part 19

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Relates:[ link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5274)

## Description
Change the type of credentials, from type 4 to type 9.

## Minimum version of Cortex XSOAR
- [x] 6.5.0

## Does it break backward compatibility?
  - [x] No

### CODE REVIEW

**Comment by maimorag** [comment]
credentials don't work for WhatIsMyBrowser [link for open issue](https://jira-hq.paloaltonetworks.local/browse/CRTX-85714) therefore, the build fails.

**Comment by Shellyber** [comment]
@maimorag API key seems to be updated. Please check if it works and remove the test from skipped. The build should pass afterward.

**Comment by Shellyber** [comment]
@maimorag What's the status here?

File: Packs/WhatIsMyBrowser/Integrations/WhatIsMyBrowser/WhatIsMyBrowser.py [hunk 1]
```diff
@@ -19,8 +20,11 @@
 
 
 def http_request(data):
+    api_key = demisto.params().get('credentials_api_key', {}).get('password') or demisto.params().get('api_key')
+    if not api_key:
+        return_error('API key must be provided.')
```
Comment (Shellyber) [inline Packs/WhatIsMyBrowser/Integrations/WhatIsMyBrowser/WhatIsMyBrowser.py:25]: Raise exception instead of return_error

**Review by Shellyber — APPROVED** [review]

**Pull request merged**